tracer/test/test-applications/Samples.Shared/SampleHelpers.cs

```diff
@@ -9,6 +9,9 @@ namespace Samples
     public class SampleHelpers
     {
         private static readonly Type NativeMethodsType = Type.GetType("Datadog.Trace.ClrProfiler.NativeMethods, Datadog.Trace");
+        private static readonly Type TracerType = Type.GetType("Datadog.Trace.Tracer, Datadog.Trace");
+        private static readonly MethodInfo GetTracerInstance = TracerType.GetProperty("Instance").GetMethod;
```

Comment:
Any reason why you got the property's getter method and didn't use `PropertyInfo.GetValue()`? Just curious.

Reply:
No, that's just what I've always done 😂